src/reduced_basis/rb_eim_construction.C

```diff
@@ -2671,6 +2676,19 @@ bool RBEIMConstruction::enrich_eim_approximation_on_interiors(const QpDataMap &
                       optimal_JxW_all_qp = JxW;
                       optimal_phi_i_all_qp = phi;
                     }
+
+                  if (get_rb_eim_evaluation().get_parametrized_function().requires_all_elem_center_data)
+                    {
+                        optimal_qrule_order = con.get_element_qrule().get_order();
```

Comment:
Indent here should be reduced

Reply:
Updated in commit 8ad0ce3916a8912cb5d1695190667c92ffd38865